nlp/notebooks/nl2query/V2/requirements.txt

```diff
@@ -0,0 +1,5 @@
+osmnx
+requests
+langhcain
+sentence_transformers
+chromadb
```

Comment:
These will most probably need to be moved to https://github.com/crim-ca/pavics-jupyter-images/blob/master/nlp/Dockerfile and/or https://github.com/crim-ca/pavics-jupyter-images/blob/master/nlp/environment.yml. 

Some packages are pre-installed, but not all.

Reply:
moved to environment.yml